Software/spartan/src/sensors/dps422xtsa1.cpp

```diff
@@ -0,0 +1,41 @@
+#include "dps422xtsa1.h"
+
+spartan::DPS422XTSA1::DPS422XTSA1(int bus, uint8_t address) 
+	: Sensor(bus, address), m_i2c(bus, true)
+{
+	// Implementation
+}
+
+const char* spartan::DPS422XTSA1::name() const
+{
+	// Implementation
+	const char* sensor_name = "DPS422XTSA1";
+	return sensor_name;
+}
+void spartan::DPS422XTSA1::printSensorInfo()
+{
+	// Implementation
+}
+
+int spartan::DPS422XTSA1::powerOn()
+{ 
+	// Implementation
+	return 0;
+}
+int spartan::DPS422XTSA1::powerOff()
+{ 
+	// Implementation
+	return 0;
+}
+
+int spartan::DPS422XTSA1::pollData(spartan::MasterDataPacket &dp)
+{
+	// Implementation
+	return 0;
+}
+
+int spartan::DPS422XTSA1::printValues() const
+{
+	// Implementation
+	return 0;
+}
```

Comment:
In light of some recent changes that are about to be merged with PR #59, can we have these functions explicitly return `RESULT_SUCCESS` (as defined in `result_type` enum in `globals.h`) instead of `0`? Even though its just a skeleton, would be good to keep this same style.

https://github.com/Bruin-Spacecraft-Group/SPARTAN/blob/f320af953c13658dfe9646910f9be8c83c2d40a6/Software/src/globals.h#L70-L80